tests/infra/node.py

```diff
@@ -511,6 +515,28 @@ def get_ledger(self):
 
         return current_ledger_dir, [committed_ledger_dir]
 
+    def hide_committed_ledger_chunk(self):
+        ledger_dir = self.remote.ledger_paths()[0]
+        ledger_chunks = list(pathlib.Path(ledger_dir).glob("*.committed"))
+
+        def interval(path):
+            start, end = path.stem.rsplit("_")[-1].split("-")
+            return int(start), int(end)
+
+        # Pick a random chunk, near the start of the ledger, to avoid the cache
+        (chunk_to_hide,) = random.sample(ledger_chunks[:5], 1)
```

Comment:
This is fairly nasty, I think the right change to make is to spin up a new node *without* access to historical files instead, but:

1. it's not quite the same thing (gap vs no files up to that point)
2. annoyingly, we only do reconfig with quotes enabled, not in virtual mode. We don't do any reconfig currently in e2e_logging.

We could:

1. move this to reconfig, but that's annoying because having lots of ledger files is useful, and only the logging test really does that.
2. keep this here, but then we add reconfig to logging tests.